dle/search/services.py

```diff
@@ -40,41 +76,92 @@ def highlight_text_by_term(text: str, search_term: str) -> Tuple[str, bool]:
     """
     if not text:
         return "", False
-
+    ignorable = {
```

Comment:
@ky-soren you can use one of the libraries I'm using to get a more complete list of ignorable (they call them stopwords). The one i use has 337 ignorable words in it. You can use it so:

`import gensim
from gensim.parsing.preprocessing import remove_stopwords, STOPWORDS
print(STOPWORDS)  # this will print the list of stopwords
new_text = remove_stopwords(text)  # this will return a new text with the stopwords removed`